### PR TITLE
Upgrade tc-lib-api

### PR DESCRIPTION
**Pull request by fiennyangeln**
Upgrade taskcluster-lib-api to carry Access-Control-Max-Age

Related to Bug 1442636

@djmitche please review:D

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ let api = new API({
     'listens for tasks with associated notifications and handles requests to',
     'send emails and post pulse messages.',
   ].join('\n'),
+  name:         'notify',
```
Comment (imbstack) [inline src/api.js]: I'd prefer if this was just a single space rather than aligned with subsequent values.

**Comment by fiennyangeln** [comment]
@imbstack okay modified!:D

**Comment by imbstack** [comment]
Looks great to me. I will land this and deploy it on Monday!

**Comment by djmitche** [comment]
Oh, @kritisingh1 made a PR that superseded this one, so it's done :)

**Pull request closed without merging**